scripts/benchmark_cchain_range.sh

```diff
@@ -16,7 +16,14 @@ set -euo pipefail
 : "${START_BLOCK:?START_BLOCK must be set}"
 : "${END_BLOCK:?END_BLOCK must be set}"
 
-cmd="go test -timeout=0 -v -benchtime=1x -bench=BenchmarkReexecuteRange -run=^$ github.com/ava-labs/avalanchego/tests/reexecute/c --block-dir=\"${BLOCK_DIR}\" --current-state-dir=\"${CURRENT_STATE_DIR}\" ${CONFIG:+--config=\"${CONFIG}\"} --start-block=\"${START_BLOCK}\" --end-block=\"${END_BLOCK}\" ${LABELS:+--labels=\"${LABELS}\"}"
+cmd="go test -timeout=0 -v -benchtime=1x -bench=BenchmarkReexecuteRange -run=^$ github.com/ava-labs/avalanchego/tests/reexecute/c \
+  --block-dir=\"${BLOCK_DIR}\" \
+  --current-state-dir=\"${CURRENT_STATE_DIR}\" \
+  ${CONFIG:+--config=\"${CONFIG}\"} \
+  --start-block=\"${START_BLOCK}\" \
+  --end-block=\"${END_BLOCK}\" \
+  ${LABELS:+--labels=\"${LABELS}\"} \
+  ${METRICS_ENABLED:+--metrics-enabled=\"${METRICS_ENABLED}\"}"
```

Comment:
How useful this test is without these metrics? I'm wondering if we should just always run the metrics server either way (i.e do we even want this flag at all?), the metrics server is not publicly accessible since it runs in CI so I don't understand why we would care if it was running even if we weren't using it.